server/ladder_service.py

```diff
@@ -285,8 +285,11 @@ def write_rating_progress(self, player: Player, rating_type: str) -> None:
                     "command": "notice",
                     "style": "info",
                     "text": (
-                        "The system is still learning you.<b><br><br>The "
-                        f"learning phase is {progress:.0f}% complete<b>"
+                        "The matchmaking system is calibrating your skill level; "
+                        "your first few games may be slightly more imbalanced "
+                        "as the system attempts to learn your capability as a player."
+                        "<b><br><br>"
+                        f"The learning phase is {progress:.0f}% completed<b>"
```

Comment:
What is that `<b>` tag for? If it's for making the text bold, shouldn't it come right before the text, and then shouldn't the closing tag be `</b>`?
```suggestion
                        "<br><br>"
                        f"<b>The learning phase is {progress:.0f}% completed</b>"
```